File: sentimental-mario-less/mario.py
from cs50 import get_int

# Prompt user for height
while True:
    height = get_int("Waht's the height? ")
    if 1 <= height <= 8:
        break

# Make the pyramid
i = k = j = 1
while i <= height:
    while k <= (height - i):
        print(" ", end="")
        k += 1
    while j <= i:
        print("#", end="")
        j += 1
    print()
    i += 1
    j = k = 1

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Prompt user for height
    int height;
    do
    {
        height = get_int("Waht's the height? \n");
    }
    while (height < 1 || height > 8);

    //Make the pyramid
    for (int i = 1; i <= height; i++)
    {
        for (int k = 1; k <= height - i; k++)
        {
            printf("%c", ' ');
        }
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }


}